spring-cloud-starter-stream-common-tensorflow/src/main/java/org/springframework/cloud/stream/app/tensorflow/processor/TensorflowCommonProcessorConfiguration.java

```diff
@@ -56,18 +62,14 @@
  * The {@link TensorflowOutputConverter} can be extended and customized to provide a convenient data representations,
  * accustomed for a particular model (see TwitterSentimentTensorflowOutputConverter.java)
  *
- * By default the inference result is returned in the outbound Message payload. If the saveResultInHeader property is
- * set to true then the inference result would be stored in the outbound Message header by name as set by
- * the getResultHeader property. In this case the message payload is the same like the inbound message payload.
+ * The {@link OutputMessageBuilder} defines how the computed inference score is arranged withing the output message.
  *
  * @author Christian Tzolov
  * @author Artem Bilan
  */
 @EnableConfigurationProperties(TensorflowCommonProcessorProperties.class)
 public class TensorflowCommonProcessorConfiguration implements AutoCloseable {
```

Comment:
We don't need to have here `AutoCloseable`. 
Since you declare `TensorFlowService` as a bean, Spring will take care about its own `AutoCloseable` already.

Reply:
+1